main.js

```diff
@@ -160,4 +160,199 @@ const inventory = [
+// })
+//
+// console.log(purchasedInventory2);
+
+// Opdracht 2d: Zorg ervoor dat dit aantal in het blauw wordt weergegeven op de pagina.
+// const element3 = document.getElementById("purchasedInventory");
+// element3.textContent = `Amount of purchased TV's: ${purchasedInventory2}`;
+
+//     Opdracht 2e: Geef in het rood weer hoeveel tv's er nog verkocht moeten worden.
+
+// let remainingInventory2 = 0;
+//
+// inventory.map ((remainingInventory) => {
+//   return remainingInventory2 += (remainingInventory.originalStock - remainingInventory.sold
+// )})
+//
+// console.log(remainingInventory2);
+//
+// const element4 = document.getElementById("toBeSoldTv");
+// element4.textContent = `Amount of TV's yet to be sold: ${remainingInventory2}`;
+
+// Opdracht 3a: Gebruik een array-methode om alle tv merken (zoals Philips, NIKKEI, etc.) in een lijst op de pagina weer te geven. Zorg ervoor dat dit ook zou werken als we 200 tv's in onze array zouden hebben staan. Dat er dubbele namen in zitten, is niet erg.
+
+// const productNames = inventory.map ((productName) => {
+//   return productName.brand;
+// })
+//
+// console.log(productNames);
+
+// Opdracht 3b: Schrijf de code uit 3a om naar een functie die een array met tv-objecten verwacht. Het is handig om onze scripts als functies op te zetten, zodat we ze gemakkelijk kunnen hergebruiken. Tip: vergeet deze functie -declaratie niet aan te roepen!
+
+// let outcome6 = "";
+//
+// const productnames3 = (input) => {
+//   for (let i = 0; i < inventory.length; i++) {
+//     outcome6 += input;
+//   } return outcome6;
+// }
+//
+// const element6 = document.getElementById('banaan');
+// element6.textContent = `The tv brand names are: ${productnames3(inventory[i].brand)}`;
+
+
+
+// const productNames2 = inventory.map ((productName) => {
+//   return productName.brand;
+// })
+//
+// const element6 = document.getElementById('banaan');
+// element6.textContent = `The tv brand names are: ${productNames2}`;
+
+
+// Opdracht 4a: Maak een herbruikbare functie die een string genereert voor de naam van één tv en deze teruggeeft in het format [merk] [type] - [naam] zoals Philips 43PUS6504/12 - 4K TV of NIKKEI NH3216SMART - HD smart TV.
+
+
+const productNames4 = (input2) => {
+  let outcome7;
+  for (let i = 0; i < inventory.length; i++) {
+
+    const tvBrand = inventory[input2].brand
+    const tvType = inventory[input2].type
+    const tvName = inventory[input2].name
+
+    outcome7 = tvBrand + ' ' + tvType + ' - ' + tvName;
+  } return outcome7
+}
+
+const element7 = document.getElementById('list');
+element7.textContent = ` ${productNames4(1)}`;
+
+// Opdracht 4b: Maak een herbruikbare functie die de prijs van één tv als parameter verwacht (zoals 379 of 159) teruggeeft in het format €379,- of €159,-.
+//
+
+let outcome8 = '';
+
+const productNames5 = (input3) => {
+  let outcome8;
+  for (let i = 0; i < inventory.length; i++) {
+    outcome8 = inventory[input3].price;
+  } return outcome8;
```

Comment:
De for-loop is niet nodig, omdat je niets met de `i` doet
```suggestion
  return inventory[input3].price;
```